code/parse/sexp.cpp

```diff
@@ -36649,30 +36729,30 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"arrays and pointers.\r\n\r\nPlease note that only numeric variables are supported.  Any "
 		"attempt to access a string variable will result in a value of SEXP_NAN_FOREVER being returned.\r\n\r\n"
 		"Takes 1 argument...\r\n"
-		"\t1:\tIndex of variable, from 0 to MAX_SEXP_VARIABLES - 1." },
+		"\t1:\tIndex of variable, from 0 to " SCP_TOKEN_TO_STR(MAX_SEXP_VARIABLES) " - 1." },
```

Comment:
I realize this would require adding, say, a `MAX_SEXP_VARIABLE_INDEX` `#define`... but it would make much more sense to just display `249` instead of `250 - 1`.

Reply:
Well there are a lot of uses of `SCP_TOKEN_TO_STR`.  It would be necessary to add a special `#define` for each different token.

Reply:
Yes, but by my count that's exactly two tokens, which isn't a lot of overhead compared to the user not having to do math to understand the help text.

Reply:
Ok.